Bottom: aa830536d8a2ad0dc2826412b291cf6171832ccd
Top:    3a94f2233b29c12f94a8c1ad02b1b92df3821792
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:02:49 +0000

Refresh of remove-debug-from-outline

---

diff --git a/src/polygon.c b/src/polygon.c
index 46101dc..6a6e0c2 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -2248,14 +2248,14 @@ struct clip_outline_info {
 static int
 arc_outline_callback (const BoxType * b, void *cl)
 {
-  ArcType *line = (ArcType *)b;
+  ArcType *arc = (ArcType *)b;
   struct clip_outline_info *info = cl;
   POLYAREA *np, *res;
 
 #ifdef DEBUG_CIRCSEGS
-  if (!(np = ArcPoly (line, line->Thickness)))
+  if (!(np = ArcPoly (arc, arc->Thickness)))
 #else
-  if (!(np = ArcPoly (line, ROUTER_THICKNESS)))
+  if (!(np = ArcPoly (arc, ROUTER_THICKNESS)))
 #endif
     return 0;
